framework/signature-inference-scripts/infer-and-annotate.sh

```diff
@@ -0,0 +1,97 @@
+#!/bin/sh
```

Comment:
Since there is only one script in this directory, and since other scripts are found in checker/bin/, I suggest you move this script there.